doc/rst/source/changes.rst

```diff
@@ -4,6 +4,83 @@
+
+New Modules:
+------------
+
+#. :doc:`batch`: Automate batch job processing by replicating a master script with job-specific parameters.
+#. :doc:`grdmix`: Blending and transforming grids and images, including manipulating transparency.
+#. :doc:`grdinterpolate`: Interpolate new 2-D grids or 1-D data series from a 3-D data cube.
+#. :doc:`grdgdal`: Execute GDAL raster programs (such as info, dem, grid, translate, rasterize or warp), from GMT.
+
+New Core Module Features:
+-------------------------
+
+#. :doc:`begin`: Ignore the user's *gmt.conf* files normally included by using **-C**.
+#. :doc:`gmtget`: Options **-D**, **-I**, **-N**, and **-Q** handle download and query of remote data sets.
+#. :doc:`gmtmath`: New operators **RGB2HSV** and **HSV2RGB** for color manipulation.
+#. :doc:`gmtregress`: Let **-A** also be used to limit angles considered for LMS regressions.
+#. :doc:`gmtspatial`: New directive **-Sb** computes buffers around lines (via the optional GEOS library).
+#. :doc:`gmtvector`: Add vector operator **-Tt** that translates points by given distance in given direction.
+#. :doc:`grd2kml`: New option **-W** for adding contour overlays. Also rebuilt for global grids as well as
+   to write PNG or JPG directly (depending on transparency) without going via *PostScript* conversion (only
+   required if **-W** is used).
+#. :doc:`grdcontour`: Better handling of contour file that can now have unique angles and pens per contour.
+#. :doc:`grdconvert`: Enable scaling/translation services on output with **-Z**.
+#. :doc:`grdfill`: Implement minimum-curvature spline infill with **-As**.
+#. :doc:`grdfilter`: Let filter width optionally be a grid with variable widths.
+#. :doc:`grdgradient`: Add support for ambient light in **-N**, as in **-E**, and therefore via **-I**
+   in :doc:`grdimage` and :doc:`grdview`.
+#. :doc:`grdinfo`: Now **-C** also appends registration as last output column (0 = gridline, 1 = pixel registration).
+#. :doc:`grdmath`: New operators **DAYNIGHT** (for day/night terminator), **BLEND** (blend two grids using the weights
+   from a third), **DOT** (dot product), and **RGB2HSV**, and **HSV2RGB** for color manipulations.
+#. :doc:`grdtrack`: Determine central peak in all crossections with **-F** (requires **-C**); let **-E+c** continue
+   a track if next line is a direct continuation of previous line.
+#. :doc:`pscontour`: Better handling of contour file that can now have unique angles and pens per contour.
+#. :doc:`movie`: Add **-E** for an optional title sequence (with or without fading in/out), **-K** for fade in and
+   fade out for main animation sequence, **-Sb** and **-Sf** can now take a PostScript layer instead of a script,
+   and **-P** for adding one of six progress indicators.
+#. :doc:`sample1d`: Adds a smoothing cubic spline via **-Fs**\ *p* (for a fit parameter *p*), with optional weights (**-W**).
```

Comment:
Add entry for nearneighbor -Nn saying that it calls the GDALʻs nearest neighbor algorithm

Reply:
done